tests/suite/test_requirements.py

```diff
@@ -0,0 +1,79 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache 2.0 License.
+
+import infra.ccf
+import functools
+
+from loguru import logger as LOG
+
+
+class TestRequirementsNotMet(Exception):
+    pass
+
+
+def none(func):
```

Comment:
Why do we need a decorator that doesn't do anything?

Reply:
Just to implicitly mark that the test doesn't require any specific CCF network.